throttler.go

```diff
@@ -124,6 +125,11 @@ func (t *ksmThrottler) Kick(context.Context, *gpb.Empty) (*gpb.Empty, error) {
 }
 
 func (t *ksmThrottler) listen() (*net.UnixListener, error) {
+	uriDir := filepath.Dir(t.uri)
+	if err := os.MkdirAll(uriDir, 0750); err != nil {
```

Comment:
Might be a good idea to move the `0750` to a global `const`/`var` to (a) make it more visible and (b) allow the tests to tweak it to force failure scenarios at some later date (assuming a `var`).